ORM/DataObject.php

```diff
@@ -381,7 +410,7 @@ public function __construct($record = null, $isSingleton = false, $model = null,
 		// Identify fields that should be lazy loaded, but only on existing records
 		if(!empty($record['ID'])) {
 			$currentObj = get_class($this);
-			while($currentObj != 'DataObject') {
+			while($currentObj != 'SilverStripe\ORM\DataObject') {
```

Comment:
This is going through the inheritance of  class, I think, so `isntanceof` isn't what we want